charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+# Data Portal
+
+
+## Description
+The Data Portal is a landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search, and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+Implementation of the DCP website experience. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at petabyte scale with public APIs.
+
+## In-scope
+### General
+* UX, visual, and branding design with input from the HCA brand working group
+* Participation in the HCA DCP community standards and best practices for Continuous Integration (CI), Continuous Delivery (CD), system logging via Google Analytics, etc.
```

Comment:
I didn't see a response to my earlier question. And I suggested -or- rather than -and-:

_What is meant by Participation in ... standards? (It could read like you're helping to develop a standard for example)_

_I would also probably use the phrase HCA DCP community standards **or** best practices._

Reply:
Changed to "Adhere to ... best practices".